docs/advanced-setup.md

```diff
@@ -153,14 +154,12 @@ Installation:
 Usage:
 `... --db-url mysql+pymysql://<username>:<password>@localhost:3306/<database>`
 
-
-
 ## Configure the bot running as a systemd service
 
 Copy the `freqtrade.service` file to your systemd user directory (usually `~/.config/systemd/user`) and update `WorkingDirectory` and `ExecStart` to match your setup.
 
 !!! Note
-    Certain systems (like Raspbian) don't load service unit files from the user directory. In this case, copy `freqtrade.service` into `/etc/systemd/user/` (requires superuser permissions).
+Certain systems (like Raspbian) don't load service unit files from the user directory. In this case, copy `freqtrade.service` into `/etc/systemd/user/` (requires superuser permissions).
```

Comment:
Please remove/revert unnecessary/unrelated edits in this file.

They're absolutely not related to what you're trying to document - and (for example this one) break formatting / rendering.
For this particular edit, i'm sure - but it shouldn't be the point of this PR to review the full document for potentially breaking edits not related to the task at hand - just to see if the documentation part you've added explains what was changed.

for trailing whitespaces - i'd recommend to configure vscode to exclude .md files from "trim trailing whitespace".